content/en/docs/concepts/windows/user-guide.md

```diff
@@ -215,19 +222,20 @@ tolerations:
 ### Handling multiple Windows versions in the same cluster
 
 The Windows Server version used by each pod must match that of the node. If you want to use multiple Windows
-Server versions in the same cluster, then you should set additional node labels and nodeSelectors.
+Server versions in the same cluster, then you should set additional node labels and `nodeSelector` fields`.
 
-Kubernetes 1.17 automatically adds a new label `node.kubernetes.io/windows-build` to simplify this.
-If you're running an older version, then it's recommended to add this label manually to Windows nodes.
+Kubernetes automatically adds a label,
+[`node.kubernetes.io/windows-build`](/docs/reference/labels-annotations-taints/#nodekubernetesiowindows-build)
+to simplify this.
 
 This label reflects the Windows major, minor, and build number that need to match for compatibility.
-Here are values used today for each Windows Server version.
+Here are values used for each Windows Server version:
 
-| Product Name                         |   Build Number(s)      |
-|--------------------------------------|------------------------|
-| Windows Server 2019                  | 10.0.17763             |
-| Windows Server, Version 20H2         | 10.0.19042             |
-| Windows Server 2022                  | 10.0.20348             |
+| Product Name                         | Label value(s)       |
+|--------------------------------------|----------------------|
+| Windows Server 2019                  | 10.0.17763           |
+| Windows Server, version 20H2         | 10.0.19042           |
```

Comment:
We can drop Windows Server, Version 20H2. It is no longer supported by MS

Reply:
I think that's a separate issue best covered outside this PR. A PR to fix that is likely to get reviewed and merged within a couple of weeks of opening it.

Reply:
I can do a commit for it, actually.